app/assets/scripts/components/map.js

```diff
@@ -106,7 +107,7 @@ const Map = React.createClass({
                 `<dt class='card-meta__label'>Status</dt>` +
                 `<dd class='card-meta__value card-meta__value--status'>${status}</dd>` +
                 `<dt class='card-meta__label'>Location</dt>` +
-                `<dd class='card-meta__value card-meta__value--location'>${byName(marker.region)[locationLang]}</dd>` +
+                `<dd class='card-meta__value card-meta__value--location'>${marker.type === 'district' ? byNameDist(marker.region)[locationLang] : byNameGove(marker.region)[locationLang]}</dd>` +
```

Comment:
This should be a variable declaration and use `objectPath#get` in case `marker.region` is falsy.